### PR TITLE
PTL-10818_deploy_review

### PR DESCRIPTION
**Pull request by someth2say**
Related: https://github.com/RedHatTraining/DO378/pull/1206

### CODE REVIEW

File: deploy-source/apps/expense-service/src/main/docker/Dockerfile.jvm [hunk 1]
```diff
@@ -75,7 +75,7 @@
 #   accessed directly. (example: "foo.example.com,bar.example.com")
 #
 ###
-FROM registry.ocp4.example.com:8443/ubi8/openjdk-17:1.14
+FROM registry.ocp4.example.com:8443/ubi8/openjdk-17:1.18
```
Comment (ebramirez) [inline deploy-source/apps/expense-service/src/main/docker/Dockerfile.jvm:78]: Can we update this to Java 21?

File: deploy-source/solutions/expense-service/src/main/docker/Dockerfile.jvm [hunk 2]
```diff
@@ -75,7 +75,7 @@
 #   accessed directly. (example: "foo.example.com,bar.example.com")
 #
 ###
-FROM registry.ocp4.example.com:8443/ubi8/openjdk-17:1.14
+FROM registry.ocp4.example.com:8443/ubi8/openjdk-17:1.18
```
Comment (ebramirez) [inline deploy-source/solutions/expense-service/src/main/docker/Dockerfile.jvm:78]: Same here.

**Review by ebramirez — APPROVED** [review]
Just one question, everything went smoothly.

File: deploy-source/apps/expense-service/src/main/docker/Dockerfile.jvm (hunk 1 above)
Comment (someth2say) [reply deploy-source/apps/expense-service/src/main/docker/Dockerfile.jvm:78]: I guess we can, yes.
But we should? Codestarts use `FROM registry.access.redhat.com/ubi8/openjdk-17:1.18`, that is Java 17.
I don´t see a reason to update, unless there is any benefit.

**Pull request merged**